core/util/wakelock.js

```diff
@@ -0,0 +1,199 @@
+/*
```

Comment:
This file is only used from the app, so it should probably be moved to the app directory instead.

Reply:
Done.